bundle/metadata/annotations.yaml

```diff
@@ -1,5 +1,6 @@
 annotations:
-  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.bundle.channel.default.v1: threescale-2.14
+  operators.operatorframework.io.bundle.channels.v1: threescale-2.14
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
```

Comment:
@eguzki want to tweak this value: 
```
operators.operatorframework.io.bundle.package.v1: 3scale-operator
```
here or at the community operators repo level?

Reply:
line 7 of this file btw.

Reply:
That is part of the patch that will be applied in https://github.com/redhat-openshift-ecosystem/community-operators-prod

I do not know how to apply to upstream. We would need a hard refactor that would imply changes in downstream as well. So good enough for now.

Reply:
I think we can do this upstream once we move to newer operator-sdk (it supports providing package name when building a bundle)

Reply:
In community PR we have correct values, similar to v0.10.1 - 
https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/4150/files#diff-bb64d4f16f07ef63aea32a8fa54c9cde707db9252f3961ee06f174cdc397b278

Reply:
ok, I see, it will be only in Community PR. Thanks